resources/event-bus/values.yaml

```diff
@@ -6,8 +6,8 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   event_bus:
-    dir: develop/
-    version: "36059183"
+    dir: pr/
```

Comment:
The changes in values.yaml can be in separate PR if you wanna avoid merging an image with PR tag.

Reply:
I've seen folks doing following this approach recently